pkgs/development/libraries/png++/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, libpng
+, docSupport ? true, doxygen ? null
+, testsSupport ? true
```

Comment:
@matthewbauer This turned out better then expected. The worst of ugly hacks is now gone. Thanks :)